scss/pivotgrid.scss

```diff
@@ -0,0 +1,12 @@
+@import "variables";
+@import "mixins/all";
+
+
+// Dependencies
+@import "common/all";
+@import "dialog";
```

Comment:
Does the field selector depend on the TreeView styles? If so, it should be listed, too.